main.py

```diff
@@ -40,55 +40,63 @@ def make_menu_decision():
         else:
             print("I am sorry; I do not recognize this option")
             
-# This function will add the first and last name to empData file.
+# This function will add the first and last name to empData file. 
+# This will also create and store the email
 def create_emp():
     # TODO add logging info
     with open('empData.txt', 'a') as f:
         addFirst = input("Enter the employee's first name: ")
         addLast = input("Enter the employyee's last name: ")
         newEmp = Employee(addFirst, addLast)
         f.write(f'Name: {newEmp.full_name}, Email: {newEmp.email}\n')
-        print("Name successfully added.")
+        print("Employee successfully added.")
+
+def check_emp_file():
+    emp = input("Enter the employee's name: ")
+
+    with open('empData.txt', 'r') as f:
+        if emp in f.read():
+            return(True)
```

Comment:
You don't have to specifically surround the return value in parenthesis.

return True

would suffice. In fact, if you wanted to return multiple values (a tuple) you can do the following (hypothetical)

return True, "My error message"

later on you can capture this tuple by

success_result, error_message = check_emp_file()   # your values are populated in the order of how it's returned in check_emp_file

Reply:
Thats interesting. I wanted to use this return statement as a way to get the users name. I will look into that

Reply:
I want to access the string from the emp variable for password storage function. Is that possible?

Reply:
First... I need to understand what you're trying to accomplish with the employee variable. Is this ONLY for the username? Or is it an object which contains data (username, and whatever else)?

I'll assume username for now, thinking you're trying to say...hey... I want to create a password for THIS specific username

You need to checkout parameters. In short, think of them as variables that are defined for your function. When you call create_pass(), there is nothing in the parenthesis because you aren't passing any data into it

def create_pass(username):   # you can define more than 1 parameter by separating them with a comma. 
    # blah blah blah, you can get rid of line 92 where you ask for the employee name
     f.write('Name: {username}\nSalt: {salt}\nKey: {key}')

# later on in the code you can do
my_awesome_name= input("Yo what's the employee name")

create_pass(my_awesome_name)  # you pass in the variable, or you can pass in a string "pissedOffBadger"
# Key thing to remember is, it'll get passed into the respective parameters. In this case username is parameter 1, so my_awesome_name will be username

def awesome_function(users=[], debug=False):
     if debug:
         print(users)

above we have default parameters. It means if I do not pass in a value for it... it will default to whatever I defined. Because these parameters have default values we can also specify which value goes to when calling this method.

awesome_function(debug=True, users=["sammy","smeagle"])  # we don't have to go "in order" like before

Reply:
Sidenote You can comment a function or anything by using the triple quotes

```python
def create_pass(username):
"""
Prompts user for a password which gets assigned to `username`
:param username: Employee Name
"""
# code for function
```